index.js

```diff
@@ -40,11 +41,46 @@ function defaultFormatter(messages) {
   return output.trim();
 }
 
+// load custom rules
+function loadCustomRules(options) {
+  let rulesDir = options.rulesDir.replace(/\\/g, '/');
+  if (fs.existsSync(rulesDir)) {
+    if (fs.statSync(rulesDir).isDirectory()) {
+      rulesDir += /\/$/.test(rulesDir) ? '' : '/';
+      rulesDir += '**/*.js';
+      glob.sync(rulesDir, {
+        dot: false,
+        nodir: true,
+        strict: false,
+        silent: true
+      }).forEach(file => {
+        loadRule(file, options);
+      });
+    } else {
+      loadRule(rulesDir, options);
+    }
+  }
+}
+
+function loadRule(filepath, options = {}) {
+  filepath = path.resolve(filepath);
+  try {
+    const ruleObj = require(filepath); // eslint-disable-line import/no-dynamic-require
+    const ruleOption = options[ruleObj.id]; // we can pass a value to the rule
+    ruleObj.rule(HTMLHint, ruleOption);
+  } catch (err) {
+    throw new Error(err.message);
```

Comment:
Is this try catch necessary? By wrapping it like this you'll lose the stack trace etc